config/rbac/role.yaml

```diff
@@ -0,0 +1,28 @@
+
+---
```

Comment:
This was newly added by the `update-generated-crd-code.sh` script after rebasing - I'm not sure quite what's up with it (the other yaml updates are all very clearly follow from the controller-gen update, but not this)

Reply:
I can assert this is happening with the current master branch and am looking into it. I think we should ignore it for this PR, and if anything needs to be adjusted we'll do it separately.